### DISCUSSION TITLE
VM gets stuck on empty screen (with UTM logo) after rebooting from Windows Installer

### DISCUSSION THREAD

**Discussion by shaperones0** [utmapp/UTM#6128] [Windows] [answered]
After installing Windows 11 (downloaded iso from CrystalFetch for apple silicon chip), and rebooting from Installer (everything went normal, reached the restart prompt at the end), VM gets stuck on empty screen with UTM logo on it.

**Configuration**  
* UTM Version: 4.1.5 (74)
* macOS Version: 12.7.1
* Mac Chip (Intel, M1, ...): M1

**Debug log**  

[debug.log](https://github.com/utmapp/UTM/files/14218271/debug.log)

**Comment by osy** [comment]
Sometimes that happens and you just have to manually restart it.

**Comment by shaperones0** [comment, marked as answer]
upgrading to newest version fixed the issue
